### PR TITLE
Tl2

### PR DESCRIPTION
**Pull request by nickgrok**
This is the new method under activity_nick_grokhowsky_2.py.

### CODE REVIEW

**Comment by nickgrok** [comment]
Hi Anna, I am sorry I re-opened this again.  It looks like I am getting a linter error that line 38 is too long.  I don't know how to shorten this line because of the python conventions we need to follow, and I did run black on this code.  Is this the nesting issue you mentioned in previous requests I made?  Also, to make it easier the code on line 38 is (formatted correctly in the actual code):

*** I just found the comment you made to use r.recode ***  I will attempt to apply it to this problem.  

gs.mapcalc(
        "out = if(out < -2.58, -3, if(out >= -2.58 & out < -1.96, -2, if(out >= -1.96 & out < -1.65, -1, if(out >= -1.65 & out <= 1.65, 0, if(out > 1.65 & out < 1.96, 1, if(out > 1.96 & out <= 2.58, 2, if(out > 2.58, 3, null())))))))",
        env=env,
    )

I could probably write a new line for each if statement, but I'm not sure if that would fix it.

**Comment by nickgrok** [comment]
I tried using r.recode but the documentation for the rules parameter is not very clear.  Also, I looked at the r.reclass function, but it wasn't clear either.  

I tried a few methods, such as: 

rules= - <<EOS   *-2.58:-3:-3 ... EOS
rules = rules.txt (this file contained *-2.58:-3:-3 ...)
rules = "rules.txt"
rules = -*-2.58:-3:-3
rules = *-2.58:-3:-3
rules = "*-2.58:-3:-3"

At this point I am probably going to just leave the original TL, but I am curious if there is better documentation somewhere?

**Comment by petrasovaa** [comment]
> I tried using r.recode but the documentation for the rules parameter is not very clear. Also, I looked at the r.reclass function, but it wasn't clear either.
> 
> I tried a few methods, such as:
> 
> rules= - <<EOS *-2.58:-3:-3 ... EOS
> rules = rules.txt (this file contained _-2.58:-3:-3 ...) rules = "rules.txt" rules = -_-2.58:-3:-3
> rules = _-2.58:-3:-3 rules = "_-2.58:-3:-3"
> 
> At this point I am probably going to just leave the original TL, but I am curious if there is better documentation somewhere?

I can see this could be tricky, either you can create a file with the rules, or you can use write_command with stdin parameter:
```
rules = ["*:-2.58:-3:-3",
         "-2.58:-1.96:-2:-2",
         ... ,
        "2.58:*:3:3"]
gs.write_command("r.recode", input=out, output="out", rules="-", stdin="\n".join(rules), env=env)
```

File: activities/activity_nick_grokhowsky_2.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        "r.recode",
+        input="out",
+        output="out",
+        rules="-",
+        stdin="\n".join(rules),
+        env=env,
+    )
+
+    # Change colors for high and low maps
+    gs.run_command("r.colors", map="out", color="elevation", env=env)
```
Comment (petrasovaa) [inline activities/activity_nick_grokhowsky_2.py]: differences color ramp would be better here

File: activities/activity_nick_grokhowsky_2.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+- All gs.run_command/read_command/write_command/parse_command
+  need to be passed env parameter (..., env=env)
+"""
+
+import grass.script as gs
+
+
+# Function to show significantly high and low pixels
+def run_significantValues(elev, env, **kwargs):
```
Comment (petrasovaa) [inline activities/activity_nick_grokhowsky_2.py]: TL requires 'scanned_elev' as parameter, not 'elev'.

**Review by petrasovaa — CHANGES_REQUESTED** [review]
This PR deletes/modifies files you should not be touching.

**Comment by wenzeslaus** [comment]
The code submitted and merged in #21 seems more fitting to the task, so closing this one. Thanks for trying to fix it anyway.

**Pull request closed without merging**